components/Tabs/tests/snapshot/TabBarView/MDCTabBarViewSnapshotTests.m

```diff
@@ -1212,4 +1212,82 @@ - (void)testCustomView {
   [self generateSnapshotAndVerifyForView:self.tabBarView];
 }
 
+#pragma mark - rectForItem:inCoordinateSpace:
+
+- (void)testRectForItemInCoordinateSpaceWithinBounds {
+  // Given
+  self.tabBarView.items = @[ self.item1, self.item2, self.item3 ];
+  CGSize intrinsicContentSize = self.tabBarView.intrinsicContentSize;
+  self.tabBarView.bounds =
+      CGRectMake(0, 0, intrinsicContentSize.width, intrinsicContentSize.height);
+  UIView *itemRectOverlayView = [[UIView alloc] init];
+  itemRectOverlayView.backgroundColor =
+      [UIColor.magentaColor colorWithAlphaComponent:(CGFloat)0.25];
```

Comment:
nit: Is there a reason you cast to CGFloat rather than using `0.25f`?

Reply:
Same for the others below.

Reply:
Yep, we don't use `0.123f` in our codebase. We decided to use double-precision and stick with it. There are a few stragglers after a massive change we did, but we even do things like

```
float f = (float)0.25;
```

Forcing the downcast makes it easier to remember that we're retaining full-width precision on 64-bit hosts.  Occasionally it has made a difference in computed values.